Integrations/integration-SumoLogic.yml

```diff
@@ -66,6 +66,11 @@ configuration:
   defaultvalue: ""
   type: 0
   required: false
+- display: Time between fetches (in seconds)
+  name: fetchDelay
+  defaultvalue: ""
```

Comment:
Add default value
The name might be tricky here, because if the system time will be configured to be bigger then it will work by that